news/4546.trivial

```diff
@@ -0,0 +1 @@
+Move attributes from RequirementSet to Resolver
```

Comment:
Yeah, I had seen that in the sources.

I figured writing these isn't any effort really so I'll keep on doing this.